src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -2,15 +2,16 @@ import { ThemeProvider as EmotionThemeProvider } from '@emotion/react';
 import { css, Global } from '@emotion/react';
 import emotionReset from 'emotion-reset';
 import { ThemeSwitchProvider, useThemeSwitch } from 'hooks/useThemeSwitch';
-import { keys, merge, pick } from 'lodash';
+import keys from 'lodash/keys';
+import merge from 'lodash/merge';
+import pick from 'lodash/pick';
```

Comment:
99.99% sure this is unnecessary and `import { keys, merge, pick } from 'lodash';` already treeshakes

Reply:
i run the analyzer and it was exporting all lodash before this

Reply:
TIL lodash exports in commonjs.
in that case maybe we can use this https://www.npmjs.com/package/@optimize-lodash/rollup-plugin

Reply:
nice one i will check it out, seems that it solves the issue